libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -657,6 +683,14 @@ void CompilerUtils::convertType(Type const& _typeOnStack, Type const& _targetTyp
 
 void CompilerUtils::pushZeroValue(Type const& _type)
 {
+	if (auto const* funType = dynamic_cast<FunctionType const*>(&_type))
+	{
+		if (funType->location() == FunctionType::Location::Internal)
+		{
+			m_context << m_context.errorTag();
```

Comment:
Ah, now I understand, you need to initialize array elements that contain internal functions, with some values.